en/docs/tutorials/create-your-first-scheduled-task.md

```diff
@@ -0,0 +1,218 @@
+
+!!! info
+    You are saving the scheduled task in a Choreo-maintained repository as specified in [Step 1.1: Create a new project and add a scheduled task component](#step-11-create-a-new-project-and-add-a-scheduled-task-component).
+
+1. Click **Sync with Choreo Upstream** in the bottom panel of the page (highlighted in red). In the message that appears, click **Sync my changes with Choreo**.
+
+    ![Sync with Choreo upstreeam](../assets/img/tutorials/scheduled-task/sync-with-choreo-upstream.png){.cInlineImage-full}
+
+2. In the left panel, enter a commit message (e.g., `Implement Scheduled Task`) and click on the tick.
+
+    ![Commit message](../assets/img/tutorials/scheduled-task/commit-message.png){.cInlineImage-full}
+
+    Select **Yes** in the message that appears to specify that you need to stage the changes.
+
+3. To push the changes to the private GitHub repository maintained by Choreo, click **0↓ 1↑** in the bottom panel.
+
+    !!! tip
+        This link appears only after the system completes the committing process.
+
+    ![Push changes](../assets/img/tutorials/scheduled-task/push-changes.png){.cInlineImage-full}
+
+    Then click **OK** in the message that appears to confirm.
+
+Once the Web Editor has successfully pushed the changes to the GitHub repository, it indicates this by displaying the text **In sync with Choreo upstream** for the `main.bal` file as shown below.
+
+![In sync](../assets/img/tutorials/scheduled-task/in-sync-with-choreo-upstream.png){.cInlineImage-full}
+
+## Step 2: Deploy
+
+ Once you have developed a scheduled task and synced its configuration to the Choreo Console, you need to deploy it to be able to run it. To deploy it, follow this procedure:
+
+1. Click the **Deploy** icon. Then in the **Build Area** card, click **Deploy**.
+
+    ![Deploy scheduled task](../assets/img/tutorials/scheduled-task/deploy-scheduled-task.png){.cInlineImage-full}
+
+    As a result, the **Configure & Deploy** panel appears on the right of the page.
+ 
+2. To run the scheduled task every two minutes, enter `2` in the **Every** field. 
+
+3. Click **Deploy** in the **Configure & Deploy** panel.
+
+Once the deployment process is complete, a message appears to notify you of it. From the time you deploy a scheduled task, it starts running continuously, retrieving the population of the USA every two minutes until you undeploy it.
+
+You can proceed to the next section to observe the scheduled task.
+## Step 3: Observe
+
+You can observe the scheduled task to verify whether it runs every two minutes as expected and check whether it successfully retrieves the specified data.
+
+To open the Observability statistics, click **Observe** in the **Development** card. Alternatively, you can click the **Observe** icon**.
+
+The logs section displays the logs generated every two minutes with population statistics of the USA. In addition, the graphs visualize the throughput and latency statistics.
```

Comment:
Does this get auto refreshed? or should I manually do it? Is it realtime data? or periodic? Shall we mention that?

Reply:
Added a tip.